bitnami/elasticsearch/values-production.yaml

```diff
@@ -243,7 +251,7 @@ data:
 ingest:
   enabled: true
   name: ingest
-  replicas: 2
+  replicas: 3
```

Comment:
Why this change?

Reply:
We are increasing the number of nodes from 2 to 3 in the case of `master` and `data`, why not for `coordinating` and `ingest`?